builders/substrate/interfaces/features/governance/parameters.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Parameters Pallet
+description: Learn how Moonbeam's Parameters Pallet safely and dynamically modifies network config items via on-chain governance, removing the need for runtime upgrades.
+keywords: democracy, substrate, pallet, moonbeam, polkadot, voting, vote, referenda, parameters
+---
+
+# The Parameters Pallet
+
+## Introduction {: #introduction }
+
+The Parameters Pallet on Moonbeam is a governance-focused module that enables community-driven modifications to key network configuration items, such as the deposit requirement for the randomness module, directly on-chain through proposals and votes, without necessarily requiring a runtime upgrade. By removing the need for frequent code changes, the Parameters Pallet allows the network to respond more quickly to evolving needs while maintaining transparency and consensus.
+
+This guide will provide an overview of the extrinsics, storage methods, and getters for the pallet constants available in the Parameters Pallet on Moonbeam. This guide assumes you are familiar with governance-related terminology; if not, please check out the [governance overview page](/learn/features/governance/#opengov){target=_blank} for more information.
+
+## Parameters Pallet Interface {: #parameter-pallet-interface }
+
+### Extrinsics {: #extrinsics }
+
+The Parameters Pallet provides one extrinsic (functions):
+
+???+ function "**setParameter**(keyValue) - Sets the value of a parameter"
```

Comment:
```suggestion
???+ function "**setParameter**(keyValue) - sets the value of a parameter"
```

Can you please check the rest of the guide? The first word after the hyphen should not be capitalized